build.env

```diff
@@ -16,14 +16,14 @@ BASE_IMAGE=quay.io/ceph/ceph:v17
 CEPH_VERSION=quincy
 
 # standard Golang options
-GOLANG_VERSION=1.17.10
+GOLANG_VERSION=1.18.5
```

Comment:
Go 1.19, is a major release. Generally, Major releases are considered stable because they undergo complete testing.
Maybe we should use it?

Reply:
we stick to N-1 release (at least for now) if we all are agreed, we can move to 1.19

cc @ceph/ceph-csi-maintainers